tests/e2e/performance/csi_tests/test_pvc_bulk_creation_deletion_performance.py

```diff
@@ -227,7 +229,27 @@ def test_bulk_pvc_creation_deletion_measurement_performance(
         full_results.add_key("bulk_pvc_deletion_time", total_deletion_time)
 
         # Write the test results into the ES server
-        full_results.es_write()
+        if full_results.es_write():
+            res_link = full_results.results_link()
+            log.info(f"The Result can be found at : {res_link}")
+
+            # Create text file with results of all subtest (4 - according to the parameters)
+            self.write_result_to_file(res_link)
+
+    def test_bulk_pvc_creation_deletion_results(self):
```

Comment:
This is in same test right, can we combine to one test at the end of the script, something similar to https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/performance/io_workload/test_fio_benchmark.py#L598, this will avoid duplication in code as well as the test name

Reply:
since it different data to display, it should be different "test name" as well.

Reply:
Will change the implementation.